.env

```diff
@@ -0,0 +1 @@
+DB_ENV=testing
```

Comment:
This is BAD. Make sure to add this to git ignore too. If you were to add api keys or a secret, you would expose it to the public. Make sure this is hidden so that no one trying your code makes that mistake either.